crates/holochain_state/src/test_utils.rs

```diff
@@ -363,3 +315,52 @@ pub fn dump_db(txn: &Transaction) {
     let stmt = txn.prepare("SELECT * FROM DhtOp").unwrap();
     dump(stmt);
 }
+
+#[cfg(test)]
+mod tests {
+    use holochain_sqlite::error::DatabaseResult;
+    use holochain_sqlite::rusqlite::Transaction;
+
+    fn _dbg_db_schema(db_name: &str, conn: &Transaction) {
+        #[allow(dead_code)]
+        #[derive(Debug)]
+        pub struct Schema {
+            pub ty: String,
+            pub name: String,
+            pub tbl_name: String,
+            pub rootpage: u64,
+            pub sql: Option<String>,
+        }
+
+        let mut statement = conn.prepare("select * from sqlite_schema").unwrap();
+        let iter = statement
+            .query_map([], |row| {
+                Ok(Schema {
+                    ty: row.get(0)?,
+                    name: row.get(1)?,
+                    tbl_name: row.get(2)?,
+                    rootpage: row.get(3)?,
+                    sql: row.get(4)?,
+                })
+            })
+            .unwrap();
```

Comment:
Probably passable for tests, but named values would be be better here incase the schema changes

Reply:
That'd be a sqlite change, I imagine that's pretty stable on these kinds of things.

I agree but this code is just moved, not added